.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: build
+
+on:
+  push:
+    branches: [main, '[0-9].[0-9]']
+  pull_request:
+    branches-ignore: ['*docs', '*noci']
+
+
+jobs:
+  build:
+    name: default build job
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: install dependencies
+        run: sudo .install/install_script.sh
+      - name: install python packages
```

Comment:
Take control of the version of python you receive with the setup-python action. This way it doesn't change under the covers.